vsintegration/src/vs/FsPkgs/FSharp.LanguageService/servicem.fs

```diff
@@ -2016,12 +2021,32 @@ type FSharpPackage() as self =
     let callback = new ServiceCreatorCallback(CreateIfEnabled)
     
     let mutable mgr : IOleComponentManager = null
+
+    let fsharpSpecificProfileSettings =
+        [| "TextEditor.F#.Insert Tabs", box false
+           "TextEditor.F#.Brace Completion", box true
+           "TextEditor.F#.Indent Style", box 1u |]
     
     override self.Initialize() =
         UIThread.CaptureSynchronizationContext()
+        self.EstablishDefaultSettingsIfMissing()
         (self :> IServiceContainer).AddService(typeof<FSharpLanguageService>, callback, true)
         base.Initialize()
 
+    /// In case custom VS profile settings for F# are not applied, explicitly set them here.
+    /// e.g. 'keep tabs' is the text editor default, but F# requires 'insert spaces'.
+    /// We specify our customizations in the General profile for VS, but we have found that in some cases
+    /// those customizations are incorrectly ignored.
+    member private this.EstablishDefaultSettingsIfMissing() =
```

Comment:
@enricosada - I actually factored as a function originally, but `Package.GetService` is a protected method, and the F# compiler won't let you invoke it from a curried func, even in a derived class. So implementing as a member was required.

I will add null check.  This service should be available in all SKUs with F# support , but can't hurt to be defensive.